services/ui-src/src/components/pages/GetStarted/ReportGetStartedPage.tsx

```diff
@@ -190,7 +190,7 @@ const sx = {
     fontWeight: "bold",
   },
   image: {
-    maxWidth: "20rem",
+    maxWidth: "18rem",
```

Comment:
This makes the layout with the image feel unbalanced when there's two columns in larger views. I would do:
```
maxWidth: "20rem",
width: "100%",
```
And then change https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/a6247dbc9153b19209281a826c0ca62a7e7423d4/services/ui-src/src/components/pages/GetStarted/ReportGetStartedPage.tsx#L67

to

```
<Box width="100%">
```